stacks/dhis2-core/helmfile.yaml

```diff
@@ -34,6 +34,8 @@ releases:
       - commonLabels:
           im: "true"
           im-default: "true"
+          im-deployment-id: "{{ requiredEnv "DEPLOYMENT_ID" }}"
+          im-instance-id: "{{ requiredEnv "INSTANCE_ID" }}"
           im-id: "{{ requiredEnv "INSTANCE_ID" }}"
```

Comment:
Should we keep both `im-id` and `im-instance-id`, given that they will be the same?